openupgradelib/openupgrade.py

```diff
@@ -2814,6 +2814,13 @@ def format_message(f):
             with savepoint(env.cr):
                 # Strange artifact found in a filter
                 domain = f.domain.replace("%%", "%")
+                if verbose:
+                    logger.info(
+                        "Probing filter '%s' on model %s with domain %s",
```

Comment:
```suggestion
                        "Checking filter '%s' on model %s with domain %s",
```

`Probing` is not a valid gerund: https://translate.google.com/details?sl=en&tl=es&text=probing&op=translate

Reply:
done